File: DroidsAPI/src/main/java/com/github/rochedo098/droidsapi/api/item/ComponentType.java
package com.github.rochedo098.droidsapi.api.item;

public enum ComponentType {
    PROCESSING_UNIT("Processing Unit (PU)"),
    TEMPORARY_MEMORY("Temporary Memory (TM)"),
    DISK_DRIVE("Disk Drive (DD)");

    private final String id;

    ComponentType(String id) {
        this.id = id;
    }

    public String toString() {
        return this.id;
    }
}

File: src/main/kotlin/com/github/rochedo098/droids/api/item/ComponentType.kt
package com.github.rochedo098.droids.api.item

enum class ComponentType(private val id: String) {
    PROCESSING_UNIT("Processing Unit (PU)"), TEMPORARY_MEMORY("Temporary Memory (TM)"), DISK_DRIVE("Disk Drive (DD)");

    override fun toString(): String {
        return id
    }
}